cos/models/Player.py

```diff
@@ -140,6 +140,8 @@ def perform_turn_option(self, turn_option, game, data):
             self.buy_settlement(settlement_to_buy, game.game_board)
             self.remove_resources_for_settlement()
             self.remaining_settlements -= 1
+            if self.remaining_settlements == 0:
+                game.winner = self.name
```

Comment:
Probably should set this to the player object, not just the name. That way the whole player object can be retrieved easily